Cargo.toml

```diff
@@ -152,4 +152,5 @@ rpath = false
 large_futures = "warn"
 
 [workspace.lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ["cfg(tarpaulin)"] }
```

Comment:
is this related to force_hash_collisions change, or something separate?

Reply:
I think it allows `tarpaulin` to be used as a `feature` even if the crate doesn't define it